pkg/api/norman/customization/cluster/validator.go

```diff
@@ -746,6 +748,29 @@ func (v *Validator) validateGKEConfig(request *types.APIContext, cluster map[str
 	return nil
 }
 
+// validateGKENetworkPolicy performs validation around setting enableNetworkPolicy on GKE clusters which turns on Project Network Isolation
+func (v *Validator) validateGKENetworkPolicy(cluster map[string]interface{}, clusterSpec *v32.ClusterSpec, prevCluster *v3.Cluster) error {
```

Comment:
`cluster` and `clusterSpec` are two forms of the same data, you could just choose one or the other.

Reply:
I'll make this change